LoopKit/DosingDecisionStore.swift

```diff
@@ -9,7 +9,6 @@
 import os.log
 import Foundation
 import CoreData
-import HealthKit
```

Comment:
[nit] Not relevant to this PR, but okay.

Reply:
ah, I can remove it if you feel strongly...it's cruft from trying to debug the problem I was having primarily.

Reply:
No worries.